docs/en/source/installation/driver.rst

```diff
@@ -0,0 +1,77 @@
+*************************************************
+NPU Kernel Driver and Firmware Installation
+*************************************************
+
+FuriosaAI provides the kernel driver and firmware for Alveo U250 and AWS F1 evaluation environments.
```

Comment:
```suggestion
Currently, FuriosaAI provides the kernel driver and firmware of Alveo U250 and AWS F1.
```

Mentioning 'evaluation environment' in Korean document wasn't necessary. I'll reflect this change to Korean document too.